todo_app/data/todo_items.py

```diff
@@ -0,0 +1,31 @@
+import collections
+from turtle import title
+from typing import Collection
+from unittest import result
+import requests
+import json
+import os
+import pymongo
+from dotenv import load_dotenv
+load_dotenv()
+
+mongo_connection=(os.getenv('MONGO_CONNECTION_STRING'))
+mongo_db=(os.getenv('MONGO_DB_NAME'))
+
+client = pymongo.MongoClient(mongo_connection)
+database = client['todoapp-DB']
+
+def get_items():  
+  collection = database['todoapp-Collection']
+  documents = list(collection.find())
+  print(documents)
+  return documents
+
+def add_item(title,description):
+  collection = database['todoapp-Collection']
+  collection.insert_one({"title":title,"description":description,"status":"Not Started"})
+
+def update_item(id):
+  collection = database['todoapp-Collection']
+  collection.update_one({"_id":id},{"$set":{"status":"Complete"}})
```

Comment:
Minor here, but _some_ versions of MongoDB (not ours) will fail here - technically the id stored is an "ObjectId" type, but we're using a string. We can fix that by:
* Adding `from bson import ObjectId` to the top of the file
* Converting the id to an ObjectId before we pass it to Mongo, e.g.:
```suggestion
  collection.update_one({"_id": ObjectId(id)},{"$set":{"status":"Complete"}})
```

Reply:
Updated